pkgs/development/python-modules/opencensus/default.nix

```diff
@@ -0,0 +1,53 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, python
+, unittestCheckHook
+, google-api-core
+}:
+
+let
+  opencensus-context = buildPythonPackage rec {
+    pname = "opencensus-context";
+    version = "0.1.3";
+
+    checkInputs = [ unittestCheckHook ];
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "a03108c3c10d8c80bb5ddf5c8a1f033161fa61972a9917f9b9b3a18517f0088c";
+    };
+  };
```

Comment:
This must be moved to python packages.

Reply:
It could be, but still `opencensus-context` is a build buildinput of `opencensus`. Install both packages will cause a conflict, since they were supposed to be installed in the same path `site-packages/opencensus`.
So even if `opencensus-context` becomes a independent package, `opencensus` should still use it as a buildinput, and users must not install it along with `opencensus-context`.

Reply:
oh, then we should better not expose it.